<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AGNSS_L76K_Dialog</class>
 <widget class="QDialog" name="AGNSS_L76K_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" columnstretch="2,1">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>FTP Client</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="FTP_Client" name="ftp_client" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Reference Position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="9" column="0">
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>fAcc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_pAcc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QCheckBox" name="checkBox_useCurPos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Use Current Position</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_wn">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>tAcc</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TOW</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Data Type</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_position">
           <property name="text">
            <string>Position</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_time">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkBox_freqBias">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>FreqBias</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_llaFormat">
           <property name="text">
            <string>LLA format</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_fAcc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>fAcc</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkBox_Alt_Invalid">
           <property name="text">
            <string>Alt invalid</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>checkBox_freqBias</zorder>
        <zorder>checkBox_fAcc</zorder>
        <zorder>checkBox_position</zorder>
        <zorder>checkBox_time</zorder>
        <zorder>checkBox_llaFormat</zorder>
        <zorder>checkBox_Alt_Invalid</zorder>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>pAcc</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>FreqBias</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_alt">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_10">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>TimerSource</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>WN</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_fAcc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QPushButton" name="pushButton_transfer">
        <property name="text">
         <string>Transfer</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_timerSource">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_long">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_freqBias">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_lat">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_tAcc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox_restartMode">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cold start</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Warm start</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hot start</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QCheckBox" name="checkBox_useCurTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Use Current</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_tow">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lineEdit_pAcc</zorder>
     <zorder>lineEdit_freqBias</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_fAcc</zorder>
     <zorder>label_7</zorder>
     <zorder>label_6</zorder>
     <zorder>lineEdit_timerSource</zorder>
     <zorder>label_8</zorder>
     <zorder>label_10</zorder>
     <zorder>lineEdit_tAcc</zorder>
     <zorder>label</zorder>
     <zorder>label_3</zorder>
     <zorder>checkBox_useCurPos</zorder>
     <zorder>lineEdit_wn</zorder>
     <zorder>label_4</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>lineEdit_alt</zorder>
     <zorder>label_9</zorder>
     <zorder>pushButton_transfer</zorder>
     <zorder>lineEdit_long</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_lat</zorder>
     <zorder>comboBox_restartMode</zorder>
     <zorder>checkBox_useCurTime</zorder>
     <zorder>lineEdit_tow</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>Data File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="3">
       <widget class="QProgressBar" name="progressBar_download">
        <property name="value">
         <number>24</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_filePath"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_selectFile">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FTP_Client</class>
   <extends>QWidget</extends>
   <header location="global">ftp_client.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
